ACLPManager/manager
# DI-20844 - Beta feedbacks and zero state time duration issue fix

## Description 📝
Addressed the beta UI feedbacks by adding tooltips, publishing resource selection on close and the zero state of time duration

## Changes  🔄

1. Added tooltips for widget level filters, icons
2. Publish resources selected only on close and clear
3. While moving back to zeroth state, make time duration to Last 30 Minutes.
4. CamelCase for aggregate function label inside the widgets
5. Adjust legend row padding and height according to number of rows

## Target release date 🗓️
08-10-2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-10-03 at 10 03 01 AM](https://github.com/user-attachments/assets/45de1db9-5b9b-49cd-82a4-f0a86d330d27)|![Screenshot 2024-10-03 at 10 03 06 AM](https://github.com/user-attachments/assets/aff3474f-fefc-4b05-bc52-4489139576ef)|
|![Screenshot 2024-10-03 at 10 03 11 AM](https://github.com/user-attachments/assets/be6d5a95-7a11-47fa-8c52-ddf215755225)|![Screenshot 2024-10-03 at 10 03 13 AM](https://github.com/user-attachments/assets/292c9590-2d11-41b3-8e63-583a4e5f2321)|
|![Screenshot 2024-10-03 at 10 03 54 AM](https://github.com/user-attachments/assets/44748617-a71b-41c1-b125-317bee620465)||

## How to test 🧪

1. Login as mock and navigate to monitor tab
2. Select any dashboard and all required filters
3. You should be able to test all features

## As an Author I have considered 🤔

*Check all that apply*
